cpp/modules/deck.gl/core/test/shaderlib/project/viewport-uniforms-test.cpp

```diff
@@ -27,9 +27,76 @@
 using namespace mathgl;
 using namespace deckgl;
 
-TEST(ViewportUniforms, Simple) {
+/// \brief Fixture for testing viewport-uniforms implementation.
+class ViewportUniformsTest : public ::testing::Test {
+ protected:
+  ViewportUniformsTest() {}
+};
+
+TEST_F(ViewportUniformsTest, Simple) {
   // TODO(isaac@unfolded.ai): Add tests
 
   // TODO(isaac@unfolded.ai): Check for non-finite outputs
   // TODO(isaac@unfolded.ai): Check for coordinate origin (copy from JS)
 }
+
+// Ensures viewport uniform values are correct by comparing them to a known expected output.
+// Input and output values taken from web-based deck.gl Flight Paths example.
+TEST_F(ViewportUniformsTest, WebMercator) {
+  WebMercatorViewport::Options opts;
+  opts.width = 640;
+  opts.height = 480;
+  opts.longitude = 7.0;
+  opts.latitude = 47.65;
+  opts.zoom = 4.5;
+  opts.pitch = 50.0;
+  opts.bearing = 0.0;
+
+  auto viewport = std::make_shared<WebMercatorViewport>(opts);
+  auto uniforms = getUniformsFromViewport(viewport, 1, Matrix4<double>::MakeUnit(), COORDINATE_SYSTEM::DEFAULT,
+                                          Vector3<double>{}, false);
+
+  auto viewProjectionMatrix = Matrix4<double>{0.1060660171779,
```

Comment:
Is there a comment that disables clang formatting for such cases? Similar to `// prettier-ignore` in JS?

Reply:
Good call. These got formatted just before I pushed so I didn't get to deal with them. Added `// clang-format off/on` instructions